2022/wot-cg-2022.html

```diff
@@ -0,0 +1,297 @@
+    </title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="//www.w3.org/StyleSheets/TR/base">
+    <style>
+      body {
+        max-width: 60em;
+        margin: auto;;
+      }
+      *:target {
+        background-color: yellow;
+      }
+      li {
+        margin-bottom: 9pt;
+      }
+      .note {
+        background-color: yellow;
+        padding: 10px;
+      }
+      .remove {
+        background-color: yellow;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>
+      [DRAFT] Web of Things Community Group Charter
+    </h1>
+    <p>
+      <span class="remove">{TBD: remove next sentence before submitting for
+      approval}</span> This Charter is work in progress. To submit feedback,
+      please use <span class="remove">https://github.com/w3c/charter-drafts/issues</span> Issues or 
+      the <span class="remove">https://github.com/w3c/charter-drafts/pull/392</span> Pull Requst 392
+      where Charter is being developed.
+    </p>
+    <ul>
+      <li>This Charter: <span class="remove">https://github.com/w3c/charter-drafts/blob/gh-pages/2022/wot-cg-2022.html</span>
+      </li>
+      <li>Previous Charter: <span class="remove">https://www.w3.org/community/wot/charter/</span>
+      </li>
+      <li>Start Date: <span class="remove">01 June 2022 (tentative)</span>
+      </li>
+      <li>Last Modifed: <span class="remove">03 May 2022</span>
+      </li>
+    </ul>
+    <h2 id="goals">
+      Goals
+    </h2>
+    <p>
+      The goal of WoT Community Group is to accelerate the community activities around the recommendations and notes published by the WoT WG and IG. More concretely, the group aims to:
+      <ul>
+        <li>Increase awareness around the WoT specifications</li>
+        <li>Collect use cases from the wider community</li>
+        <li>Collect implementation experience and technologies from the wider community</li>
+        <li>Facilitate the implementation of the specifications by providing help in form of guidelines and tutorials</li>
+      </ul>
+    </p>
+    <h2 id="scope-of-work">
+      Scope of Work
+    </h2>
+    <p>
+      In order to achieve the above goals, the scope of the Web of Things Community Group includes the following activities:
+      <ul>
+        <li>Organizing events such as hackathons, ideathons, etc.</li>
+        <li>Supporting other communities to host W3C WoT related events, i.e. satellite events</li>
+        <li>Increasing communication between WoT-related developers through platforms such as Gitter, Matrix, StackOverflow, etc.</li>
+        <li>Increasing the social media presence of the W3C WoT</li>
+        <li>Reporting the results of its activities as a group back to the Web of Things Working/Interest Group, the W3C membership, and the Web community</li>
+        <li>Handling questions and discussions on the standards of the Web of Things Working/Interest Group</li>
+      </ul>
+    </p>
```

Comment:
```suggestion
        <li>Increasing the social media presence of the W3C WoT</li>
        <li>Handling questions and discussions on the standards of the Web of Things Working/Interest Group</li>
      </ul>
    <p>
    In addition,
    the WoT CG will report the results of its activities back to the Web of Things Working and Interest Groups,
    the W3C membership, and the Web community.
    </p>
```

Reply:
Reporting" seemed a bit out of place in the list.  Also minor HTML fixes (technically lists can't be inside paragraphs).